### PR TITLE
Add some information about tags in the build instructions

### PR DESCRIPTION
**Pull request by O1LER**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,10 +65,11 @@ If you are building for ARM (like aarch64), you need to run `export VCPKG_FORCE_
 You can build on **Windows, Linux** or other platforms by following these steps:
 
 1. Check out the repository with git: `git clone --recursive https://github.com/BeamMP/BeamMP-Server`.
-2. Go into the directory `cd BeamMP-Server`.
-3. Run CMake `cmake -S . -B bin -DCMAKE_BUILD_TYPE=Release` - this can take a few minutes and may take a lot of disk space and bandwidth.
-4. Build via `cmake --build bin --parallel --config Release -t BeamMP-Server`.
-5. Your executable can be found in `bin/`.
+2. Specify the server version to build by cheking out a tag: `git checkout tags/v3.4.1` - See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags
```
Comment (WiserTixx) [inline README.md]:
```suggestion
2. Specify the server version to build by checking out a tag: `git checkout tags/v3.4.1` - [Possible versions/tags](https://github.com/BeamMP/BeamMP-Server/tags)
```
small spelling bit and tidied up the link

**Review by Mack29446 — APPROVED** [review]
lgtm :)

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ If you need support with understanding the codebase, please write us in the Disc
 
 ## About Building from Source
 
-We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.1.0`. 
+We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags, as well as https://github.com/BeamMP/BeamMP-Server/releases to check which version is marked as a release/prerelease. We recommend to use the latest release https://github.com/BeamMP/BeamMP-Server/releases/latest
```
Comment (WiserTixx) [inline README.md]: Could the links here be put in [text](url) format please?
For example:
```suggestion
We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See https://github.com/BeamMP/BeamMP-Server/tags for possible versions/tags, as well as https://github.com/BeamMP/BeamMP-Server/releases to check which version is marked as a release/prerelease. We recommend to use the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest)
```

**Review by WiserTixx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (O1LER) [reply README.md]: Nice

File: README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ If you need support with understanding the codebase, please write us in the Disc
 
 ## About Building from Source
 
-We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.1.0`. 
+We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See [the tags](https://github.com/BeamMP/BeamMP-Server/tags) for possible versions/tags, as well as [the releases](https://github.com/BeamMP/BeamMP-Server/releases) to check which version is marked as a release/prerelease. We recommend to use the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest)
```
Comment (WiserTixx) [inline README.md]:
```suggestion
We only allow building unmodified (original) source code for public use. `master` is considered **unstable** and we will not provide technical support if such a build doesn't work, so always build from a tag. You can checkout a tag with `git checkout tags/TAGNAME`, where `TAGNAME` is the tag, for example `v3.4.1`. See [the tags](https://github.com/BeamMP/BeamMP-Server/tags) for possible versions/tags, as well as [the releases](https://github.com/BeamMP/BeamMP-Server/releases) to check which version is marked as a release/prerelease. We recommend using the [latest release](https://github.com/BeamMP/BeamMP-Server/releases/latest).
```

**Review by WiserTixx — CHANGES_REQUESTED** [review]

**Review by WiserTixx — APPROVED** [review]

**Pull request merged**